frontend/frontend/src/app/add-projects/add-projects.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="add_project_bg">
+<div class="add_project_bg" *ngIf="roles=='Sales Team'">
```

Comment:
please change values to camelcase "salesTeam" instead of "Sales Team"

Reply:
updated variable name in camelCase